src/node/rpc/base_endpoint_registry.h

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "ds/json.h"
+#include "endpoint_registry.h"
+#include "node/quote.h"
+#include "node/rpc/node_interface.h"
+
+namespace ccf
+{
+  enum class QuoteFormat
```

Comment:
FYI, until https://github.com/microsoft/CCF/issues/1468 is complete, we're still using (what OE refers to) as the legacy quote format (see https://github.com/openenclave/openenclave/blob/master/include/openenclave/attestation/sgx/evidence.h#L83-L123). I think we should upgrade sooner than later (I've got a half-finished branch somewhere to use the new API).

I don't think this impacts this enum though, as CCF v1 should just choose a quote format and stick with it.

Reply:
So this enum is just trying to capture what CCF did to generate the quote. I assume OE's format tag is somewhere inside the quote blob? Or should we also be extracting that and passing it around to support later verification?

Reply:
Not all OE quote formats are tagged (the newer ones are), but CCF must have its own separate tag because the quote may or may not come from OE.